src/aiida/cmdline/commands/cmd_bug_report.py

```
@@ -0,0 +1,310 @@
+                LOGGER.warning(f'Failed to close broker while collecting bug report diagnostics: {exception}')
+
+    return {'connected': connected, 'status': status}
+
+
+def _check_daemon() -> dict[str, Any]:
+    from aiida.manage import get_manager
+
+    """Return daemon connection information for the current profile."""
+    try:
+        status = get_manager().get_daemon_client().get_status()
+    except Exception as exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)